x-pack/tasks/build.js

```diff
@@ -4,20 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import del from 'del';
 import { resolve } from 'path';
 import { writeFileSync } from 'fs';
 import pluginHelpers from '@kbn/plugin-helpers';
 import { ToolingLog } from '@kbn/dev-utils';
 import { generateNoticeFromSource } from '../../src/dev';
 
+async function moveFiles(gulp, src, dest) {
```

Comment:
https://nodejs.org/docs/latest/api/fs.html#fs_fs_rename_oldpath_newpath_callback

Reply:
I've answered in the main comment I left below 😃